### ISSUE TITLE
Unfortunately Telegram is not accessible for blind people

### ISSUE THREAD

**Issue by discapacidad5** [telegramdesktop/tdesktop#10328]
Unfortunately Telegram is not accessible for blind people, the desktop version I would like you to be able to work on improving the sensitivity so that Telegram can be read by screen readers is a necessary demand for people with visual disabilities who want to migrate to Telegram but are with the difficulty that unfortunately the desktop client is not read by the screen reader

**Comment by discapacidad5** [comment]
These are some of the many pages that exist of people demanding accessibility in Telegram


https://www.reddit.com/r/Telegram/comments/9kl6ee/blind_people_are_almost_entirely_shut_out_of/

https://nvda.groups.io/g/nvda/topic/accessible_telegram/4072117

https://www.change.org/p/telegram-should-be-accessible-for-visually-impaired

https://www.gitmemory.com/issue/telegramdesktop/tdesktop/476/478961956

**Comment by ilya-fedin** [comment]
#476
